src/patterns/data/default/entity_attribute.tsv

```diff
@@ -7241,3 +7241,15 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
 OBA:1001090	feather color	UBERON:0000022	feather	PATO:0000014	color
 OBA:1001091	egg shell trait	UBERON:0005079	eggshell	PATO:0000001	quality
 OBA:1001092	egg shell color	UBERON:0005079	eggshell	PATO:0000014	color
+OBA:2040142	head width	PATO:0000921	width	UBERON:0000033	head
+OBA:2040143	metafemur length	PATO:0000122	length	HAO:0001140	metafemur
```

Comment:
We should probably add Uberon IDs for all OBA terms as @cmungall suggests on slack (uberon channel)

Reply:
The HAO classes are not vertebrate anatomy terms though.